base/docs/basedocs.jl

```diff
@@ -1431,7 +1431,7 @@ Unsigned
 """
     Bool <: Integer
 
-Boolean type, containing the values `true` and `false`.
+Boolean type, containing the values `true` (`== 1`) and `false` (`== 0`) .
```

Comment:
```suggestion
Boolean type, containing the values `true` and `false`.
Note that `Bool` is kind of number and `false` is numerically
equal to `0` and `true` is numerically equal to `1`.
Moreover, `false` and acts as a multiplicative "strong zero":

    julia> false == 0
    true

    julia> true == 1
    true

    julia> 0 * NaN
    NaN

    julia> false * NaN
    0.0
```

Reply:
Very nice! Small typo - missing "a" in "Note that `Bool` is kind of number"